src/main/java/com/google/devtools/build/lib/packages/ImplicitOutputsFunction.java

```diff
@@ -161,7 +173,7 @@ public SkylarkImplicitOutputsFunctionWithMap(ImmutableMap<String, String> output
    */
   public abstract static class SafeImplicitOutputsFunction extends ImplicitOutputsFunction {
     @Override
-    public abstract Iterable<String> getImplicitOutputs(AttributeMap map);
+    public abstract Object getImplicitOutputs(AttributeMap map); /* guaranteed to output Iterable<String> */
```

Comment:
If it's guaranteed to be Iterable<String>, why can't it be the type of the return value?